.buildkite/pipeline.yml

```diff
@@ -341,6 +341,17 @@ steps:
       config: cpu
       queue: central
       slurm_ntasks: 1
+  
+  - label: ":computer: Rising Bubble hybrid invariant total energy"
+    key: "cpu_rising_bubble_hybrid_invariant_total_energy"
+    command:
+      - "julia --color=yes --project=examples examples/hybrid/bubble2d_invariant_totalenergy.jl"
+    artifact_paths:
+      - "examples/hybrid/output/bubble2d_invariant_etot/*"
+    agents:
+      config: cpu
+      queue: central
+      slurm_ntasks: 1
 
   - label: ":computer: Rising Bubble 3D hybrid invariant"
     key: "cpu_rising_bubble_3d_hybrid_invariant"
```

Comment:
conflicting keys in 357 and 368

Reply:
Additional information on prognostic vars added to keys